java/arcs/android/devtools/DevWebServerImpl.kt

```diff
@@ -63,6 +71,9 @@ object DevWebServerImpl : DevWebServer, NanoWSD("localhost", 33317) {
                 send("Socket Open")
                 ping(PING_PAYLOAD)
                 open = true
+                onOpenSocketCallbacks.forEach { callback ->
+                    send(callback())
```

Comment:
Ah, so it does send the text. I feel like a better API design here would be to pass the callback some interface that lets it decide what to do when a new socket is opened. If you wanted to keep this pattern, I think a name other than "callback" would be better, also - String may be too general.  Does it being a String mean that any client needs to do their own JSON/Proto/etc serialization?

Reply:
Doh! Updated to be () -> Unit.